DEV/org.openl.core/src/org/openl/gen/groovy/GroovyInterfaceImplGenerator.java

```diff
@@ -53,11 +64,7 @@ protected String generateClassDescription() {
                 implementationChain.append(", ");
             }
         }
-        StringBuilder description = new StringBuilder(super.generateClassDescription()).append(" ")
-            .append(IMPLEMENTS)
-            .append(" ")
-            .append(implementationChain);
-        return description.toString();
+        return super.generateClassDescription() + " " + IMPLEMENTS + " " + implementationChain;

```

Comment:
How many times `IMPLEMENTS` constant is used? Maybe it is better to inline it?
Also, `implementationChain` can be prefixed by `.append(super.generateClassDescription()).append(" implements ");`